test_scripts/WebEngine/WebEngineProjection/016_Registration_disallowed_app_revoked_permissions_in_PT.lua

```diff
@@ -0,0 +1,60 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Check that App will be disallowed to register with HMI type WEB_VIEW 
+-- when application has revoked permissions in policy table
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WebEngine device is connected to SDL
+-- 3. PT contains record for App1 with null permissions
+
+-- Sequence:
+-- 1. Application1 try to register with WEB_VIEW appHMIType
+--  a. SDL reject registration of application (resultCode: "DISALLOWED", success: false)
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.defaultMobileAdapterType = "WS"
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local appHMIType = { "WEB_VIEW" }
+local appsRAIParams = {
+  appHMIType = appHMIType,
+  syncMsgVersion = {
+    majorVersion = 7,
+    minorVersion = 0
+  }
+}
+
+local function updatePreloadedPT(pAppId, pAppHMIType)
+  local preloadedTable = common.getPreloadedPT()
+  local appId = config["application" .. pAppId].registerAppInterfaceParams.fullAppID
+  local appPermissions = common.cloneTable(preloadedTable.policy_table.app_policies.default)
+  appPermissions.AppHMIType = pAppHMIType 
+  appPermissions.enabled = true
+  appPermissions.transportType = "WS"
+  appPermissions.hybridAppPreference = "CLOUD"
```

Comment:
I would propose to remove redundant steps
```suggestion

```
This comment relates to next scripts too.

Reply:
@VjKlepikov fixed in https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/32/commits/fdc211806e06f5d64f56270664d8bf53f243196a